src/Analyzer.Reports.UnitTests/TestCases.cs

```diff
@@ -21,6 +21,7 @@ public class TestCases
                     new MockEvaluation
                     {
                         RuleId = "TEST-000001",
+                        RuleName = "Rule000001",
```

Comment:
I would have written it more as `Name000001` so that it is more clear from the RuleId field, but not a significant issue that you need to change.